ui/README.md

```diff
@@ -0,0 +1,64 @@
+# UI
```

Comment:
Is it safe to merge all your changes to the master branch? Doesn't it break the existing implementation?

Reply:
For the ui directory code: it's certainly safe to merge the ui directory in, because there's no direct connection between that directory and the Django app's directory. You have to compile the React app and move it (either manually or with `tools/ui buildtoae`) into /app for the /ui code to even end up on AppEngine.
For the Django code: so far I haven't made any changes to the Django code that should affect the default behavior (and none of that's out for review yet).

Reply:
Thanks, sounds good. I guess your plan for Django code changes is to still submit them into the master branch and guard them with USE_REACT_UI so that it doesn't affect production until it's ready?

Reply:
Yep. I'm going to do some refactoring along the way (to avoid code duplication), but all the code that changes behavior will be flag-guarded.